pkgs/os-specific/linux/r8152-udev-rules/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "r8152-udev-rules";
+  version = finalAttrs.src.rev;
+  src = fetchFromGitHub {
+    owner = "wget";
+    repo = "realtek-r8152-linux";
+    rev = "v2.16.3.20221209";
+    sha256 = "sha256-RaYuprQFbWAy8CtSZOau0Qlo3jtZnE1AhHBgzASopSA=";
+  };
+  dontBuild = true;
+  dontConfigure = true;
```

Comment:
```suggestion

```
if those don't break the build, please keep them enabled

Reply:
We're not building the kernel module, just using the udev rules. Taking this out will build the out-of-tree kernel module.

Reply:
Then we should at a short comment to remember that knowledge in the future.